paasta_tools/kubernetes_tools.py

```diff
@@ -1519,14 +1520,21 @@ def filter_nodes_by_blacklist(
     ]
 
 
-def maybe_add_yelp_prefix(attribute: str,) -> str:
-    return YELP_ATTRIBUTE_PREFIX + attribute if "/" not in attribute else attribute
+def paasta_prefixed(attribute: str,) -> str:
+    # discovery attributes are exempt for now
+    discovery_attributes = ["region", "superregion", "ecosystem", "habitat"]
```

Comment:
`paasta_prefixed()` is called a lot, `discovery_attributes` is initialized on every call, and then scanned linearly.
Maybe convert the list into the set and make `discovery_attributes` a global variable?